core/adapters/quotient_test.go

```diff
@@ -0,0 +1,102 @@
+package adapters_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"chainlink/core/adapters"
+	"chainlink/core/internal/cltest"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestQuotient_Perform_Success(t *testing.T) {
+	tests := []struct {
+		name   string
+		params string
+		json   string
+		want   string
+	}{
+		{"string", `{"dividend":100}`, `{"result":"1.23"}`, "81.30081301"},
```

Comment:
Is it a business requirement to accept both strings and floats for the adapters?  I notice the same pattern is used in the Multiplier adapter

Reply:
I copied Multiplier adapter in this case.

However, I think the reason is probably because JavaScript has issues storing large integers losslessly and often uses strings instead, so it makes sense to accept them here.